app/controllers/api/flash_cards_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::FlashCardsController < ApplicationController
+  MAX_AMOUNT_OF_FLASH_CARDS_TO_DISPLAY = 10
```

Comment:
How will this affect future decks? Will it limit the maximum possible flashcards in one deck to 10 or are we changing this in upcoming features when we want to return an entire deck and not only a single card?

Reply:
we will change this in future features, needed this to meet the requirements for this features tasks